recipes/r-rlist/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rlist
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.4.6.1"
+
+source:
+  fn: rlist_0.4.6.1.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/rlist_0.4.6.1.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/rlist/rlist_0.4.6.1.tar.gz
+
+
+  # You can add a hash for the file here, like md5 or sha1
+  # md5: 49448ba4863157652311cc5ea4fea3ea
+  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: testthat, stringdist, pipeR
+requirements:
+  build:
+    - r-base
+    - r-xml
+    - r-data.table
+    - r-jsonlite
+    - r-yaml
+
+  run:
+    - r-base
+    - r-xml
+    - r-data.table
+    - r-jsonlite
+    - r-yaml
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('rlist')" # [not win]
+    - "\"%R%\" -e \"library('rlist')\"" # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://renkun.me/rlist, https://github.com/renkun-ken/rlist, https://renkun.me/rlist-tutorial
+  license: MIT + file LICENSE
+  summary: Provides a set of functions for data manipulation with list objects, including mapping,
+    filtering, grouping, sorting, updating, searching, and other useful functions. Most
+    functions are designed to be pipeline friendly so that data processing with lists
+    can be chained.
+  license_family: MIT
+
+# The original CRAN metadata for this package was:
+
+# Package: rlist
```

Comment:
We should fix the skeleton generator, but if you feel like it, please clean up the comments - not strictly needed.